docs/contributing/technical/resolve-merge-conflicts.md

```diff
@@ -82,9 +81,9 @@ It will look like this:
 
 ![review merge conflicts](../../../static/img/contributing-resolve-merge-conflicts-review-conflicts.png)
 
-## Resolve conflicts
+## Resolve Conflicts
 
-Since this pull request does not modify the `package.json` file it is safe to fast-forward the changes from `origin/main`:
+Since this pull request does not modify the `package.json` file, it is safe to fast-forward the changes from `origin/main`:
```

Comment:
We might want to consider giving a more general example and instruction of resolving code-related conflicts.